tests/subsys/zigbee/osif/serial/serial_async_api/src/main.c

```diff
@@ -324,48 +331,6 @@ void test_main(void)
 	/* Enable zigbee callbacks implemented in zigbee_scheduler mock. */
 	zigbee_enable();
```

Comment:
Is there anything against moving this `zigbee_enable()` and those `printk` to testsuite setup function `configure_test_suite`? Thanks to this you could remove `test_main` function from this test.

Reply:
Good idea, already applied :)